brace-expansion module

Fresh installation gives me this error

```
> node build/dev-server.js

Listening at http://localhost:8080

webpack built c7fecbbc8f83a1bcf66e in 459ms
Unhandled rejection Error: Cannot find module 'brace-expansion'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/storage/projects/foo/node_modules/minimatch/minimatch.js:10:14)
    at Module._compile (module.js:397:26)
    at Object.Module._extensions..js (module.js:404:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/storage/projects/foo/node_modules/readdirp/readdirp.js:5:18)
    at Module._compile (module.js:397:26)
    at Object.Module._extensions..js (module.js:404:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)

Hash: c7fecbbc8f83a1bcf66e
```

Edit: I'm using vue-cli `2.1.0`

--- a/template/config/index.js
+++ b/template/config/index.js
@@ -9,8 +9,10 @@ module.exports = {
     assetsSubDirectory: 'static',
     assetsPublicPath: '/',
     productionSourceMap: true,
-    // gzip off by default as many popular static hosts such as
-    // surge or netlify already gzip all static assets for you
+    // Gzip off by default as many popular static hosts such as
+    // Surge or Netlify already gzip all static assets for you.
+    // Before setting to `true`, make sure to:
+    // npm install --save-dev compression-webpack-plugin
     productionGzip: false,
     productionGzipExtensions: ['js', 'css']
   },

--- a/template/package.json
+++ b/template/package.json
@@ -48,7 +48,6 @@
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.8.4",
     "function-bind": "^1.0.2",
-    "compression-webpack-plugin": "^0.3.1",
     "html-webpack-plugin": "^2.8.1",
     "http-proxy-middleware": "^0.12.0",
     "json-loader": "^0.5.4",

